diff --git a/node_modules/electron-notarize/lib/notarytool.js b/node_modules/electron-notarize/lib/notarytool.js
index ed59c82..441b656 100644
--- a/node_modules/electron-notarize/lib/notarytool.js
+++ b/node_modules/electron-notarize/lib/notarytool.js
@@ -71,8 +71,9 @@ function notarizeAndWaitForNotaryTool(opts) {
                 ...authorizationArgs(opts),
                 '--wait',
                 '--output-format',
+                !opts.noS3Acceleration ? '' : '--no-s3-acceleration',
                 'json',
-            ];
+            ].filter(Boolean);
             const result = yield spawn_1.spawn('xcrun', notarizeArgs);
             if (result.code !== 0) {
                 try {
